ballerina/googlesheets_client.bal

```diff
@@ -545,4 +542,54 @@ public isolated client class GoogleSheetsClient {
         }
     }
 
+    private isolated function sendRequest(http:Client httpClient, string path) returns json | error {
+        http:Response|error httpResponse = httpClient->get(path);
+        if httpResponse is http:Response {
+            int statusCode = httpResponse.statusCode;
+            json | error jsonResponse = httpResponse.getJsonPayload();
+            if jsonResponse is json {
+                error? validateStatusCodeRes = self.validateStatusCode(jsonResponse, statusCode);
+                if (validateStatusCodeRes is error) {
+                    return validateStatusCodeRes;
+                }
+                return jsonResponse;
+            } else {
+                return self.getSpreadsheetError(jsonResponse);
+            }
+        } else {
+            return self.getSpreadsheetError(<json|error>httpResponse);
+        }
+    }
+
+    private isolated function validateStatusCode(json response, int statusCode) returns error? {
+        if statusCode != http:STATUS_OK {
+            return self.getSpreadsheetError(response);
+        }
+    }   
+    isolated function getSpreadsheetError(json|error errorResponse) returns error {
+        if errorResponse is json {
+                return error(errorResponse.toString());
+        } else {
+                return errorResponse;
```

Comment:
Formatting issue here

Reply:
Updated in [9b568a4](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/54/commits/9b568a4c991b6fb0e50445d5d1ee8e690d7e64f8)